### PR TITLE
AT_02.07.004 | <Homepage> Verify the 'Create a job' link in the 'Start building your software project' section

### PR DESCRIPTION
**Pull request by julian-ole**
https://trello.com/c/sJKj30qP/899-at0207004-homepage-verify-the-create-a-job-link-in-the-start-building-your-software-project-section

### CODE REVIEW

**Review by lenashlyakhovets — APPROVED** [review]

File: cypress/e2e/homepageCreateAJobLink.cy.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/// <reference types="cypress"/>
+
+describe("Homepage > Main Panel > 'Create a job' link", () => {
```
Comment (Rom-T) [inline cypress/e2e/homepageCreateAJobLink.cy.js:3]: Please name the describe as the name of your file.

File: cypress/e2e/homepageCreateAJobLink.cy.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  it('AT_02.07.004 | <Main Panel> Verify the "Create a job" link in the "Start building your software project" section', function () {
+    cy.get('.content-block a[href="newJob"]')
+      .should("be.visible")
+      .should("have.text", "Create a job")
```
Comment (Rom-T) [inline cypress/e2e/homepageCreateAJobLink.cy.js:7]: When we make several assertions for 1 element, only the first assertion should, and all subsequent ones are written as `and`

Please move `"Create a job"` in the fixtures

File: cypress/e2e/homepageCreateAJobLink.cy.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      .click();
+    cy.url().should("include", "/newJob");
```
Comment (Rom-T) [inline cypress/e2e/homepageCreateAJobLink.cy.js:9]: Please move `"/newJob"` in the fixtures

File: cypress/e2e/homepageCreateAJobLink.cy.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    cy.get(".add-item-name .h3").should("have.text", "Enter an item name");
```
Comment (Rom-T) [inline cypress/e2e/homepageCreateAJobLink.cy.js:10]: Please move `"Enter an item name"` in the fixtures

File: cypress/e2e/homepageCreateAJobLink.cy.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress"/>
+
+import homepageMainPanel from "../fixtures/homepageMainPanel";
+
+describe("Homepage > 'Create a job' link", () => {
+  it('AT_02.07.004 | <Main Panel> Verify the "Create a job" link in the "Start building your software project" section', function () {
+    cy.get('.content-block a[href="newJob"]').should("be.visible");
+    and("have.text", homepageMainPanel.linkName).click();
```
Comment (Rom-T) [inline cypress/e2e/homepageCreateAJobLink.cy.js:8]: It should be like this: `cy.get('.content-block a[href="newJob"]').should("be.visible").and("have.text", homepageMainPanel.linkName).click();`

File: cypress/e2e/homepageCreateAJobLink.cy.js (hunk 2 above)
Comment (Rom-T) [reply cypress/e2e/homepageCreateAJobLink.cy.js]: It should be like this: `cy.get('.content-block a[href="newJob"]').should("be.visible").and("have.text", homepageMainPanel.linkName).click();`

**Review by SawkaQA — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Pull request merged**